resources/assets/sass/components/_footer.scss

```diff
@@ -0,0 +1,130 @@
+.c-footer {
+
+	padding: 132px 0 48px;
+	background: url('#{$path-images}/footer-bg.jpg') no-repeat top center;
+	background-size: cover;
+	position: relative;
+
+	&:after {
+
+		content: '';
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+		background: rgba(#111111, .6);
```

Comment:
Color literals like `#111111` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.6` should be written with a leading zero as `0.6`